File: JAVA/CreaditCardValidation.java
import java.util.Scanner;
public class CreaditCardValidation{
	public static void main(String[] args){
	Scanner input = new Scanner(System.in);

	String userInput;

	System.out.print("Enter creadit card: ");
	userInput = input.next();

	int length = userInput.length();
	//System.out.println(length);

	char firstNumber = userInput.charAt(0);
	char secondNumber = userInput.charAt(1);
	//System.out.println(firstNumber);
	
	if(length > 12 && length < 17){
		if (firstNumber == '4'){
		System.out.println("*******************************************");
		System.out.println("**Creadit Card Type: Visa Card");
		}
		else if (firstNumber == '5'){
		System.out.println("*******************************************");
		System.out.println("**Creadit Card Type: Master Card");
		}
		else if (firstNumber == '3' && secondNumber == '7'){
		System.out.println("*******************************************");
                System.out.println("**Creadit Card Type: American Express Card");
		}
		else if (firstNumber == '6'){
		System.out.println("*******************************************");
                System.out.println("**Creadit Card Type: Discover Card");
		}
		else{
		System.out.println("*******************************************");
		System.out.println("**Creadit Card Type: Not recognise");
		}
	System.out.println("**Creadit card Number: " + userInput);
	System.out.println("**Creadit Card digit length: " + length);

	System.out.println("**Cradit card validity status: Valid");
	}
	else{
	System.out.println("**Cradit card validity status: Invalid");
	}
	System.out.println("*******************************************");

	


	
}

}

File: PYTHON/creaditcardvalidation.py
user_input = input("Enter credit card Number: ")
length = len(user_input)
first_number = user_input[0]
second_number = user_input[1]

if 12 < length < 17:
	if first_number == '4':
		print("*******************************************")
		print("**Credit Card Type: Visa Card")
	elif first_number == '5':
		print("*******************************************")
		print("**Credit Card Type: Master Card")
	elif first_number == '3' and second_number == '7':
		print("*******************************************")
		print("**Credit Card Type: American Express Card")
	elif first_number == '6':
		print("*******************************************")
		print("**Credit Card Type: Discover Card")
	else:
		print("*******************************************")
		print("**Credit Card Type: Not recognised")
	print("**Credit card Number: " + user_input)
	print("**Credit Card digit length: " + str(length))
	print("**Credit card validity status: Valid")
else:
	print("**Credit card validity status: Invalid")
print("*******************************************")